Source/BestProjectEver/Public/Core/GameState/BPE_GameState.h

```diff
@@ -22,6 +23,11 @@ class BESTPROJECTEVER_API ABPE_GameState : public AGameState
 protected:
 
 	uint8 bAreAllEnemiesDead : 1;
+
+	int32 TopScore;
+
+	UPROPERTY(Replicated)
+	TArray<APlayerState*> TopScoringPlayers;
```

Comment:
It could be useful here to document your variables

Reply:
![image](https://user-images.githubusercontent.com/85327912/175393926-9715ce8d-0443-4b10-8ed5-77f7258dd515.png)